cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -398,20 +407,29 @@ Add Service Wizard: Completion Page
    :align: center
    :class: bordered-image
 
-   **Add Service Wizard, Page 7:** Congratulations screen, though there is still work to be done.
+   **Add Service Wizard, Page 6:** Congratulations screen, though there is still work to be done.
 
 
 Cluster Home Page: Status Tab
 -----------------------------
 
-.. figure:: ../_images/cloudera/cloudera-csd-09.png
+.. figure:: ../_images/cloudera/cloudera-csd-09a.png
    :figwidth: 100%
    :width: 800px
    :align: center
    :class: bordered-image
 
    **Cluster Home Page, Status Tab:** Showing all CDAP services running. *Gateway* is not an actual service.
 
+.. figure:: ../_images/cloudera/cloudera-csd-09b.png
+   :figwidth: 100%
+   :width: 800px
+   :align: center
+   :class: bordered-image
+
+   **Cloudera Manager Home Page:** Showing CDAP installed on the cluster as a service.
```

Comment:
is it possible to replace [this image](http://builds.cask.co/artifact/CDAP-DQB125/shared/build-3/Docs-HTML/4.0.0-SNAPSHOT/en/admin-manual/_images/cloudera-csd-09b.png) with a very similar one except with CDAP running and cluster not needing config updates/restarting?